packages/snap-store-mobx/src/Search/Stores/SearchFacetStore.ts

```diff
@@ -251,13 +264,14 @@ export class ValueFacet extends Facet {
 	};
 
 	constructor(
-		config: SearchStoreConfig | AutocompleteStoreConfig,
 		services: StoreServices,
 		storage: StorageStore,
 		facet: SearchResponseModelFacetValue,
-		facetMeta: MetaResponseModelFacet
+		facetMeta: MetaResponseModelFacet,
+		config: SearchStoreConfig | AutocompleteStoreConfig
 	) {
-		super(services, storage, facet, facetMeta);
+		super(services, storage, facet, facetMeta, config);
+		this.multiple = this.multiple;
```

Comment:
Added a duplicated line here - should be removed.

Reply:
oops! nice catch